src/jobflow_remote/cli/job.py

```diff
@@ -74,7 +76,7 @@ def jobs_list(
     hours: hours_opt = None,
     verbosity: verbosity_opt = 0,
     max_results: max_results_opt = 100,
-    sort: sort_opt = SortOption.UPDATED_ON.value,
+    sort: sort_opt = SortOption.UPDATED_ON,
```

Comment:
Switching to the Enum also breaks the CLI. I think I had tried at the time. Typer expects a string (as it would from the user, and then converts it to the value. Can we revert back these to be the Enum values? Or does it work for you?

Reply:
I think I've caught and reverted all of these, just had to add various `# type: ignore`s